src/main/java/com/yahoo/memory/AllocateDirectMap.java

```diff
@@ -132,10 +126,9 @@ public void load() {
 
   @Override
   public boolean isLoaded() {
-    final int ps = unsafe.pageSize();
     try {
       final long capacity = state.getCapacity();
-      final int pageCount = pageCount(ps, capacity);
+      final int pageCount = NioBits.pageCount(capacity);
       return (boolean) MAPPED_BYTE_BUFFER_ISLOADED0_METHOD.invoke(mbb, state.getNativeBaseOffset(),
```

Comment:
I would call it directly with `ZERO_DIRECT_BUFFER` as the first argument, without saving it in a field. With a comment that isLoaded0() if effectively static, i. e. doesn't use or modify the state of the provided MappedByteBuffer object, so it doesn't matter with which instance of ByteBuffer it is called. Same about force() and load().

Reply:
Done.  This removed the mbb field.